decals-cutout.py

```diff
@@ -17,7 +17,7 @@
 from astropy.io import fits
 
 DATABSE = "https://github.com/des-science/decals-cutouts/releases/download/v0.1/database_v0.fits"
-QUERY = 'select ra, dec from y6a2_coadd_object_summary where coadd_object_id = {objid};'
+QUERY = 'select ra, dec from {table}_coadd_object_summary where coadd_object_id = {objid};'
```

Comment:
I think it would be better to ask the user to specify the full name (i.e., y6a2_coadd_object_summary) if they are not using the default (this would mean the default is updated as well).